src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, AttributesDrawing {
+    private final String color;
+
+    Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    public void draw() {
+    }
```

Comment:
Similar to 'getArea', the 'draw' method should not have a default implementation in the abstract class. Each figure might have a different way of drawing, so this method should be abstract and implemented in the subclasses.